ioThread.c

```diff
@@ -0,0 +1,47 @@
+#include "ioThread.h"
+
+int ioThread(SceSize args, void *argp)
+{    
+    // dont do anything until we can confirm all modules are loaded
+    waitForKernel();    
+
+    // Initialize the psp-uart-library
+    pspUARTInit(UART_BAUD_RATE);
+
+    while (1) {
+        pspUARTWrite(COMMANDS_RIGHT_ANALOG);
+
+        pspUARTWaitForData(10000);
+
+        int recievedDataCount = pspUARTAvailable();
+
+        // If we recieved no data skip and request again
```

Comment:
Reading the response doesn't seem the smartest, possibly unneeded delays.

Im going to look into re-working this but wanted eyes on this change ahead of time for early feedback.